WooCommerce/Classes/ViewRelated/Bookings/Booking Details/BookingDetailsViewController.swift

```diff
@@ -0,0 +1,28 @@
+import UIKit
+import SwiftUI
+
+/// periphery: ignore - will be used after Booking list is ready
+final class BookingDetailsViewController: UIHostingController<BookingDetailsView> {
```

Comment:
Can we use the `BookingDetailsView` directly in `BookingTabView`? If so I think this hosting controller would be redundant.